CONTRIBUTING.md

```diff
@@ -166,3 +166,18 @@ $ python setup.py test
 ```
 
 You can run individual tests by providing the option `-a PATH_TO_TEST_FILE`.
+
```

Comment:
We decided we wouldn't do this here, yes, but in the distribution repo?

Reply:
I wasn't clear on where we landed, @0x00b1 ?

Reply:
I don't have a preference.

Reply:
Maybe we should leave this in CONTRIBUTING and add it to distribution too?